src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -158,22 +147,6 @@ def validate_policy_permissions(ns):
             'specify at least one: --key-permissions, --secret-permissions, '
             '--certificate-permissions')
 
-    key_allowed = [x.value for x in KeyPermissions]
-    secret_allowed = [x.value for x in SecretPermissions]
-    cert_allowed = [x.value for x in CertificatePermissions]
-
-    for p in key_perms or []:
-        if p not in key_allowed:
-            raise ValueError("unrecognized key permission '{}'".format(p))
-
-    for p in secret_perms or []:
-        if p not in secret_allowed:
-            raise ValueError("unrecognized secret permission '{}'".format(p))
-
-    for p in cert_perms or []:
-        if p not in cert_allowed:
-            raise ValueError("unrecognized cert permission '{}'".format(p))
```

Comment:
Same comment as in validate_key_ops except here you would need to change where you build key_allowed secret_allowed and cert_allowed to 

[x.vault.lower() for x in ...

Reply:
NM seems I see from your comments that enum_choice_list seems to handle this validation